app/assets/stylesheets/scaffolds.scss

```diff
@@ -242,6 +242,14 @@ body.hidden-shell {
   z-index: -1;
 }
 
+.broadcast-wrapper {
+  // TODO: [@thepracticaldev/delightful]: Remove these styles once
+  // https://github.com/thepracticaldev/dev.to/pull/8234 is merged.
+  z-index: 101; // Must be higher than the z-index of the sticky-nav.
+  position: fixed;
+  width: 100%;
+}
```

Comment:
As mentioned in the PR description, these styles can be removed once @ludwiczakpawel's PR, https://github.com/thepracticaldev/dev.to/pull/8234, is merged.

I've tried this out against his changes and we shouldn't need these styles since the default `<div>` styling will Just Work™️. Without these styles, the broadcast looks like this on an article page: 
<img width="1655" alt="Screen Shot 2020-06-04 at 4 56 07 PM" src="https://user-images.githubusercontent.com/6921610/83911705-8d9f2d00-a721-11ea-9c2a-09d6d42a6d0b.png">

So that's why I'm adding this fixed positioning. But with @ludwiczakpawel's PR changes, it will look like this, so I think we'll be able to remove most of this styling:
<img width="1349" alt="Screen Shot 2020-06-05 at 8 43 17 AM" src="https://user-images.githubusercontent.com/6921610/83911721-94c63b00-a721-11ea-9dc3-da3064e26c3a.png">

I'm

Reply:
Addressed! Added styles to handle the two different header configs (totally forgot that we had that, good catch on pointing that out, @ludwiczakpawel!):

_Sticky:_
<img width="1475" alt="Screen Shot 2020-06-08 at 9 53 18 AM" src="https://user-images.githubusercontent.com/6921610/84058367-009ce380-a96e-11ea-87a2-6e36c994fef6.png">

_Static:_
<img width="1475" alt="Screen Shot 2020-06-08 at 9 53 09 AM" src="https://user-images.githubusercontent.com/6921610/84058384-05619780-a96e-11ea-992d-a90e17ce28fc.png">